scripts/data_transfer.js

```diff
@@ -0,0 +1,95 @@
+function combinedScript() {
```

Comment:
Please add a section documenting how this script is used, expected inputs, outputs, etc.